src/store/connection.ts

```diff
@@ -24,11 +24,12 @@ const samplePGConnection: Connection = {
 interface ConnectionState {
   connectionList: Connection[];
   databaseList: Database[];
+  tableList: Table[];
```

Comment:
`tableList` should be a field of `Database`. And it's already exists.

https://github.com/sqlchat/sqlchat/blob/main/src/types/database.ts#L6